### PR TITLE
my solution

### PR DESCRIPTION
**Pull request by Pavlo000**
[DEMO LINK](https://Pavlo000.github.io/react_person/)

### CODE REVIEW

**Review by AKomashchuk — APPROVED** [review]
Гарна робота!

File: src/App.js [hunk 1]
```diff
@@ -27,23 +28,11 @@ const alex = {
...
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    {Person(misha)}
+    {Person(olya)}
+    {Person(alex)}
```
Comment (illiarerun) [inline src/App.js:35]: Let's use `JSX` syntax, like `<Component propName={propValue} />

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,28 @@
...
+        My name is
+        {` ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          I am
+          {` ${age}`}
+        </p>
+      )}
+      <p>{isMarried ? `My ${partnerSex()} ${partnerName}` : 'I am not married'}</p>
```
Comment (illiarerun) [inline src/components/Person/Person.jsx:25]: Let's not write code in one line, format it properly, also you forgot to add styles in `Person.scss`
```suggestion
      <p>
        {isMarried 
          ? `My ${partnerSex()} ${partnerName}` 
          : 'I am not married'
        }
      </p>
```

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,28 @@
...
+  partnerName,
+}) {
+  const partnerSex = () => (sex === 'm' ? 'wife' : 'husband');
+
+  return (
+    // eslint-disable-next-line react/jsx-filename-extension
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {` ${name}`}
```
Comment (illiarerun) [inline src/components/Person/Person.jsx:17]: use interpolation

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,28 @@
...
+      </h2>
+      {age && (
+        <p className="Person__age">
+          I am
+          {` ${age}`}
```
Comment (illiarerun) [inline src/components/Person/Person.jsx:22]: here also

**Review by illiarerun — CHANGES_REQUESTED** [review]

**Review by jstmpelowycz — APPROVED** [review]

**Pull request closed without merging**